src/components/Area.tsx

```diff
@@ -104,7 +104,7 @@ const Area = () => {
   const accessToken = useAccessToken();
   const meta = useMeta();
   const { areaId } = useParams();
-  const { data, error } = useAreas(parseInt(areaId ?? "0"));
+  const { data, error } = useArea(+areaId);
```

Comment:
(note to self: are you doing this right? `useArea(..)` returns an array of areas of length 1. Is this being handled correctly everywhere? Should we add a `select` in there to flatten this?)

Reply:
GET /areas always returns an array. So when fetching one area (as in Area.tsx) I just used the same api function, and added [0] on the data-element to get the first element. Probably not best practice, feel free to rewrite :)